src/zenml/cli/__init__.py

```diff
@@ -247,6 +284,46 @@
 
    zenml orchestrator delete ORCHESTRATOR_NAME
 
+Customizing your Container Registry
+-----------------------------------
+
+The container registry is where all the containers that are used by a
```

Comment:
I think we should rather say 'images' than containers... where all the images that are used by a ...